.github/workflows/main.yml

```diff
@@ -190,30 +193,32 @@ jobs:
         if-no-files-found: ignore
 
     - name: Configure (Autotools)
-      if: "! matrix.platform.cmake"
+      if: ${{ !matrix.platform.cmake }}
       run: |
         set -eu
-        rm -fr build-autotools
-        mkdir build-autotools
+        rm -fr build
+        mkdir build
         ./autogen.sh
-        # We do the CMake build with stb_image and the Autotools build without
-        set -- --disable-stb-image --prefix=$(pwd)/prefix_autotools
-        # mingw-w64-*-SDL2 doesn't have SDL_test, so only build this on Unix
-        if [ "${{ matrix.platform.shell }}" = 'sh' ]; then
-          set -- "$@" --enable-tests
-          set -- "$@" --enable-installed-tests
+        if [ "x${{ runner.os }}" = "xmacOS" ]; then
+          export LDFLAGS=-Wl,-rpath,${{ steps.sdl.outputs.prefix }}/lib
```

Comment:
I have to manually add the runpath of SDL2 because macOS' SIP ignores the `DYLD_LIBRARY_PATH` set by `libsdl-org/setup-sdl`.
If I don't do this, running the build-time tests fail with:
```
 FAIL: testimage
===============

dyld[7512]: Library not loaded: '@rpath/libSDL2-2.0.0.dylib'
  Referenced from: '/Users/runner/work/SDL_image/SDL_image/build/test/.libs/testimage'
  Reason: tried: '/Users/runner/work/SDL_image/SDL_image/build/.libs/libSDL2-2.0.0.dylib' (no such file), '/usr/local/lib/libSDL2-2.0.0.dylib' (no such file), '/usr/lib/libSDL2-2.0.0.dylib' (no such file)
FAIL testimage (exit status: 134)
```

CMake always uses runpath in the build directory, and removes it at install time.